live-examples/css-examples/values-and-units/angle.css

```diff
@@ -1,5 +1,5 @@
 #example-element {
-    background-color: lightsalmon;
+    background-color: #ff5b9b;
```

Comment:
This color still doesn't meet the minimum contrast guidelines, as the text to background contrast ratio is `2.91`.
How about the #0454aa - which meets [AAA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast#related_wcag_success_criteria) requirement with score `7.36`?

Reply:
I think this color looks good in dark theme but when we switch to light the text in black will not meet the contrast guideline's with #0454aa. What about #ffb703 it looks good in both dark and light theme.

Reply:
This color has a contrast ratio of `1.75` and even for me it's difficult to read the content:
![image](https://user-images.githubusercontent.com/100634371/219430242-81d79d78-d12a-4741-b1da-d40213e9b05e.png)

Reply:
ok suggested color applied

Reply:
@rohini-ranjanR We could still go with something else that you think would be better for both light and dark themes. To check the contrast ratio, first, inspect `#example-element` in dev tools, then add `color: white;` declaration in it(this shouldn't have any effect), and now click on the `white` dot:
![image](https://user-images.githubusercontent.com/100634371/219432546-35fc9093-1d8a-4c83-9689-c4f45970d49c.png)

The contrast ratio will be displayed at the bottom, comparing color with `background-color`.

Reply:
Thanks @NiedziolkaMichal . I am using #0118f3 here that is giving a contrast of `8.65`

![image](https://user-images.githubusercontent.com/109434545/219447990-60bfa5fb-4ac0-4bc6-a2f3-76e862055244.png)

looks like this in **dark theme** 

![image](https://user-images.githubusercontent.com/109434545/219448278-bb5acea0-cdf2-4fea-8005-491051ec1c69.png)

and this in **light theme** 
![image](https://user-images.githubusercontent.com/109434545/219448117-2fb66f93-1f2e-4f26-bacc-9447be727311.png)

Reply:
Thanks for noticing. How about also changing the color to white, so the example looks good in both themes?

Reply:
okay, thanks for your suggestions.